src/test/java/com/rednavis/vaadin/showcase/backend/jms/JmsTest.java

```diff
@@ -0,0 +1,32 @@
+package com.rednavis.vaadin.showcase.backend.jms;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.rednavis.vaadin.showcase.jms.MessageReceiver;
+import com.rednavis.vaadin.showcase.jms.MessageSender;
+import javax.jms.JMSException;
+import org.junit.jupiter.api.Test;
+
+class JmsTest {
```

Comment:
What kind of test is that? What it's purpose?

Reply:
I removed main() and test show that jms is working